File: Practica-cpp/Trabajo_practico_1/2-Expresiones/Ejercicio_9.cpp
/*
 * 9. Realize un programa que calcule el valor que toma la sigiente funcion para unos valores dados de x e y:
 */

#include<iostream>
#include<math.h>
using namespace std;
int main() {
    float x,y, resultado;
    cout<<"Ingrese el valor de x: ";cin>>x;
    cout<<"Ingrese el valor de y: ";cin>>y;
    resultado=(sqrt(x))/(pow(y,2)-1);
    cout<<"El resultado es: "<<resultado<<endl;
    return 0;
}
/*
 * 10. Escriba un programa que que calcule las soluciones de una ecuacion d esegundo grado de la forma ax²+bx+c=0
 * teniendo en cuenta que:
 * x1 = (-b+sqrt(pow(b,2)-4*a*c))/(2*a)
 * x2 = (-b-sqrt(pow(b,2)-4*a*c))/(2*a)
*/

File: Practica-java/src/Trabajo_Practico_1/_2_Expresiones/Ejercicio_9.java
package Trabajo_Practico_1._2_Expresiones;
import java.util.Scanner;
/*
 * 9. Realize un programa que calcule el valor que toma la sigiente funcion para unos valores dados de x e y:
 */

public class Ejercicio_9 {
    public static void main(String[] arg){
        float x,y, resultado;
        Scanner entrada = new Scanner(System.in);

        System.out.println("Ingrese el valor de x: ");
        x=entrada.nextFloat();
        System.out.println("Ingrese el valor de y: ");
        y=entrada.nextFloat();

        resultado= (float) ((Math.sqrt(x))/(Math.pow(y,2)-1));
        System.out.println("El resultado es: "+resultado);
    }
}
